components/feature/readerview/src/main/java/mozilla/components/feature/readerview/view/ReaderViewControlsView.kt

```diff
@@ -45,6 +45,15 @@ interface ReaderViewControlsView {
      */
     fun asView(): View = (this as View)
 
+    /**
+     * Tries to inflate the view if needed.
+     *
+     * See: https://github.com/mozilla-mobile/android-components/issues/5491
+     *
+     * @return true if the inflation was completed, false if the view was already inflated.
+     */
+    fun tryInflate(): Boolean
```

Comment:
nit: We're not doing (and aren't planning to do) anything with the returned boolean. So this also just be `maybeInflate` or `inflateIfNeeded` without a return value?

Reply:
I was using the return value only for writing out a test; Mike commented about that too. I wasn't sure what better way to get a test out of this without something more complicated by finding child views maybe?

Reply:
Yes, that's fair, it does make testing easier. I think it's fine to land as is.